framework/core/Mutation.pm

```diff
@@ -54,6 +54,49 @@ my $SUMMARY_FILE = "summary.csv";
 
 =head2 Static subroutines
 
+  Mutation::create_mml(instrument_classes, out_file)
+
+Generates an mml file, enabling all mutation operators for all classes listed
+in F<instrument_classes>.
+
+=cut
+sub create_mml {
+    @_ == 2 or die $ARG_ERROR;
+    my ($instrument_classes, $out_file) = @_;
+
+    my $OUT_DIR = Utils::get_dir($out_file);
+    my $TEMPLATE = `cat $MAJOR_ROOT/mml/template.mml` or die "Cannot read mml template: $!";
+
+    # The mutation operators that should be enabled in the mml file
+    my @ops =("AOR", "LOR","SOR", "COR", "ROR", "ORU", "LVR", "STD");
```

Comment:
I wonder if there is any better way of handling this hard coded mutation operators. If these are the default mutation operators, could not Major enable them (by default)?